### PR TITLE
fix: fixed non serializable values warning

### PR DESCRIPTION
**Pull request by al-rosenthal**
refacoted component to expect id instead of a full credential

# Summary of Changes

- refactored `CredentialDetails.tsx` to expect `credentialId` instead of a credential object

# Related Issues

[BC Wallet 2225](https://github.com/bcgov/bc-wallet-mobile/issues/2225)

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Comment by bryce-mcmath** [comment]
Is `credential` still being passed as a param to this screen from `NotificationListItem.tsx` and `chat-messages.tsx`?

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Just officially requesting changes as per my comment above

File: packages/legacy/core/App/modules/openid/screens/OpenIDCredentialOffer.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ type OpenIDCredentialDetailsProps = StackScreenProps<DeliveryStackParams, Screen
 
 const OpenIDCredentialDetails: React.FC<OpenIDCredentialDetailsProps> = ({ navigation, route }) => {
 
-
+    // FIXME: change params to accept credential id to avoid 'non-serializable' warnings
```
Comment (al-rosenthal) [inline packages/legacy/core/App/modules/openid/screens/OpenIDCredentialOffer.tsx:30]: I had a conversation with @bryce-mcmath and decided to not fix this as this feature is still being worked on

**Review by MosCD3 — APPROVED** [review]

**Review by bryce-mcmath — APPROVED** [review]
Great change! 👷🛠️

**Pull request merged**